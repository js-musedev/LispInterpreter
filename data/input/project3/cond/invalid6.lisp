(COND (F 5) (T NIL))