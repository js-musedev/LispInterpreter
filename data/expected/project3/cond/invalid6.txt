TYPE ERROR: Expected type: AnyNat at position [2] of COND function.    Actual: List[>=0]